tap_clickup/schemas/team.json

```diff
@@ -1,58 +1,123 @@
 {
     "definitions": {
         "user": {
-            "type": "object",
+            "type": ["object", "null"],
             "properties": {
-                    "id": {
-                        "type": "integer"
-                    },
-                    "username": {
-                        "type": "string"
-                    },
-                    "email": {
-                        "type": ["null", "string"]
-                    },
-                    "color": {
-                        "type": ["null", "string"]
-                    },
-                    "profilePicture": {
-                        "type": ["null", "string"]
-                    },
-                    "initials": {
-                        "type": ["null", "string"]
-                    },
-                    "role": {
-                        "type": ["null", "integer"]
-                    },
-                    "custom_role": {
-                        "type": ["null", "integer"]
-                    },
-                    "last_active": {
-                        "type": ["null", "string"]
-                    },
-                    "date_joined": {
-                        "type": ["null", "string"]
-                    },
-                    "date_invited": {
-                        "type": ["null", "string"]
-                    }
+                "id": {
+                    "type": "integer"
```

Comment:
To be safe I think we should add null here, as these refs haven't been heavily tested by users.